src/coffee/product-type-import.coffee

```diff
@@ -0,0 +1,90 @@
+_ = require 'underscore'
+Promise = require 'bluebird'
+ProductTypeImport = require 'sphere-product-type-import'
+{ExtendedLogger, ProjectCredentialsConfig} = require 'sphere-node-utils'
+package_json = require '../package.json'
+
+class ProductTypeImporter
+
+  ###
+  Prepare for importing, initialize used modules
+  ###
+  init: (argv) ->
```

Comment:
but than it would have to create ProductTypeImport object in that import function which is wrong because it should be initialized on some central place so it can be possibly used on other places